src/collar/oc_capture.lsl

```diff
@@ -263,7 +264,7 @@ default{
             else if (sToken == g_sSettingToken+"risky") g_iRiskyOn = (integer)sValue;
             else if (sToken == "auth_tempowner") g_sTempOwnerID = sValue; //store tempowner
             else if (sToken == g_sSettingToken+"info") g_iCaptureInfo = (integer)sValue;
-        } else if (iNum >= CMD_OWNER && iNum <= CMD_EVERYONE) UserCommand(iNum, sStr, kID, FALSE);
+        } else if (iNum >= CMD_OWNER && iNum <= CMD_EVERYONE || iNum == CMD_NOACCESS) UserCommand(iNum, sStr, kID, FALSE);

```

Comment:
Actually i think this line could just be ... 
```
iNum >= CMD_OWNER && iNum <= CMD_NOACCESS

Reply:
That would include CMD_BLOCKED (520) into the range. I guess that's not desirable.

Reply:
Ahhh true. Thanks! That actually did not cross my mind o.o Will merge as is.